### PR TITLE
Added Self-hosting guide

### PR DESCRIPTION
**Pull request by Ta1al**
- Added `config.example.json`
- Updated `readme.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,37 @@ Want to contribute? Great, we love that! Please take your time on [opening a new
 
 You can see all contributors and their GitHub-profiles [here](https://github.com/countr/countr/graphs/contributors).
 
+## Self-hosting
+We do not recommend self-hosting the bot, but it's always an option. To selfhost the bot yourself, you need to have:
+We do not recommend self-hosting the bot, but it's always an option. To selfhost the bot yourself, you need to have:
```
Comment (promise) [inline README.md]: Duplicate lines

File: README.md [hunk 2]
```diff
@@ -17,6 +17,37 @@ Want to contribute? Great, we love that! Please take your time on [opening a new
...
+| Property       | Default    | Description |
+|:---------------|:-----------|:------------|
+| `token`        |            | The Discord bot token | 
+| `database_uri` |            | The mlab.com-user and password-URI. | N/A |
+| `admins`       | `[]`       | An array of admins, the first one being the owner. | 
+| `prefix`       | `"c!"`     | The prefix you want the bot to use for commands. |
+| `mainGuild`    |            | The main guild the bot will be in. |
+| `color`        | `12404274` | The brand color in decimal number. |
+| `webhookUrl`   | `null`     | A webhook URL to post to at the end of every week. Response is `{ "value1": *counts this week*, "value2": *week number* }`. Maybe hook it up to a Twitter handle via IFTTT? |
+| `port`         | `1001`       | Creates an express server on `localhost:port` to show information about shards. |
```
Comment (promise) [inline README.md]: Don't document this, it's only for the live version

File: config.example.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "token": "discord.bot.token.here",
+  "database_uri": "mongodb://",
+  "admins": [ "your user id here" ],
+  "prefix": "c!",
+  "mainGuild": "your guild id here",
+  "color": 12404274,
+  "webhookUrl": null,
+  "port": 1001
```
Comment (promise) [inline config.example.json:9]: Don't document this, it's only for the live version

**Review by promise — CHANGES_REQUESTED** [review]
I've requested some changes, but everything else looks good.

File: README.md (hunk 1 above)
Comment (Ta1al) [reply README.md]: Done.

File: config.example.json (hunk 3 above)
Comment (Ta1al) [reply config.example.json:9]: Done.

File: README.md (hunk 2 above)
Comment (Ta1al) [reply README.md]: Done.

**Review by promise — APPROVED** [review]

**Pull request merged**